comp.html

```diff
@@ -0,0 +1,968 @@
+                    style="width: 436px; height: 102px; border:0; transform: scale(1); overflow:hidden;margin: 2rem 0 1rem; border-radius: 1rem;"
+                    sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                    
+                    <h3 class="secondary-txt-showcase" >@icon avatars</h3>
+                    <div class="row-av">
+                        <div class="avatar av-sm av-pink"><i class="fab fa-twitter"></i></div>
+                        <div class="avatar av-sm av-blue"><i class="fab fa-instagram"></i></div>
+                        <div class="avatar av-sm av-green"><i class="fab fa-linkedin-in"></i></div>
+                    </div>
+                    
+                    <iframe
+                    src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=panda-syntax&wt=sharp&l=css&ds=false&dsyoff=20px&dsblur=68px&wc=false&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=15px&lh=128%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522avatar%2520av-sm%2520av-pink%2522%253E%250A%2509%253Ci%2520class%253D%2522fab%2520fa-twitter%2522%253E%253C%252Fi%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522avatar%2520av-sm%2520av-blue%2522%253E%250A%2509%253Ci%2520class%253D%2522fab%2520fa-instagram%2522%253E%253C%252Fi%253E%250A%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522avatar%2520av-sm%2520av-green%2522%253E%250A%2509%253Ci%2520class%253D%2522fab%2520fa-linkedin-in%2522%253E%253C%252Fi%253E%250A%253C%252Fdiv%253E"
+                    style="width: 382px; height: 217px; border:0; transform: scale(1); overflow:hidden;margin: 2rem 0 1rem; border-radius: 1rem;"
+                    sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+
+                    <h3 class="secondary-txt-showcase" >@size avatars</h3>
+                    <div class="row-av">
+                        <img class="avatar av-sm" src="images/person-1.jpg" alt="">
+                        <img class="avatar " src="images/person-2.jpg" alt="">
+                        <img class="avatar av-lg" src="images/person-3.jpg" alt="">
+                    </div>
+
+                    <iframe
+                    src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=panda-syntax&wt=sharp&l=css&ds=false&dsyoff=20px&dsblur=68px&wc=false&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=15px&lh=128%25&si=false&es=2x&wm=false&code=%253Cimg%2520class%253D%2522avatar%2520av-sm%2522%2520src%253D%2522...%2522%2520alt%253D%2522..%2522%253E%250A%253Cimg%2520class%253D%2522avatar%2522%2520src%253D%2522...%2522%2520alt%253D%2522..%2522%253E%250A%253Cimg%2520class%253D%2522avatar%2520av-lg%2522%2520src%253D%2522...%2522%2520alt%253D%2522..%2522%253E"
+                    style="width: 445px; height: 102px; border:0; transform: scale(1); overflow:hidden;margin: 2rem 0 1rem; border-radius: 1rem;"
```

Comment:
The common styling in iframe component can be moved to a class as it's getting reused

Reply:
Thanks a lot Pooja for all your suggestions, actually for iframe I have different styling for all different styles.

Reply:
Small suggestion:
In iframe components, except height and width, the following are common as far as I noticed:
'border:0; transform: scale(1); overflow:hidden;margin: 2rem 0 1rem; border-radius: 1rem' 

So only these properties can be moved to a common class (DRY principle).

Reply:
> Small suggestion:
> In iframe components, except height and width, the following are common as far as I noticed:
> 'border:0; transform: scale(1); overflow:hidden;margin: 2rem 0 1rem; border-radius: 1rem'
> 
> So only these properties can be moved to a common class (DRY principle).

Ohh yess, right. Thank you! I did it now 😊🙌